test-apps/base-tests/tests/unit/build-waiter-test.ts

```diff
@@ -13,7 +13,7 @@ import { module, test } from 'qunit';
 
 // @ts-ignore
 import { Promise } from 'rsvp';
-import Token from '@ember/test-waiters/token';
+import Token from '@ember/test-waiters/__private__/token';
```

Comment:
how does this extra `/__private__/` works?

Reply:
https://github.com/emberjs/ember-test-waiters/pull/476/files#diff-6d28e496920bb420e41d13a83301a26ed6f19c4cec4dac3723042b1253d87181R77